stylesheets/stylesheet.css

```diff
@@ -111,8 +111,8 @@ a {
 .main-content h1, .main-content h2, .main-content h3, .main-content h4, .main-content h5, .main-content h6 {
   margin-top: 2rem;
   margin-bottom: 1rem;
-  font-weight: normal;
-  color: #159957; }
```

Comment:
That's because there is no green in Portus. 
#383e41 is a "gimp colour pick" of the darker blue used in the logo